components/resources/quizzes/langQuiz/LangResults.tsx

```diff
@@ -3,22 +3,70 @@ import SkillifyNavbar from "../shared/SkillifyNavbar";
 
 type LangResultsProps = {
   onBackClick: () => void;
+  score: { [key: string]: number };
 };
 
-const LangResults = ({ onBackClick }: LangResultsProps) => {
+const LangResults = ({ onBackClick, score }: LangResultsProps) => {
+  const maxLangScore = (score: { [key: string]: number }) => {
+    let maxScore = 0;
+    let preferredLang = "";
+    for (const [key, value] of Object.entries(score)) {
+      if (value > maxScore) {
+        maxScore = value;
+        preferredLang = key;
+      }
+    }
+    return preferredLang;
+  };
+
+  const preferredLang = maxLangScore(score);
+
+  type resultData = {
+    body: string;
+    src: string;
+    alt: string;
+    lang: string;
+    size: string;
+  };
+
+  const results: { [key: string]: resultData } = {
+    JavaScript: {
+      body: "The first coding language you should learn is...",
+      src: "/images/quiz/languages-quiz/javascript.png",
+      alt: "JavaScript",
+      lang: "JavaScript",
+      size: "scale-75",
+    },
+    Python: {
+      body: "The first coding language you should learn is...",
+      src: "/images/quiz/languages-quiz/python.png",
+      alt: "Python",
+      lang: "Python",
+      size: "scale-50 h-80",
+    },
+    "HTML/CSS": {
+      body: "The first coding languages you should learn are...",
+      src: "/images/quiz/languages-quiz/htmlcss.png",
+      alt: "HTML/CSS",
+      lang: "HTML/CSS",
+      size: "scale-75",
+    },
+  };
+
+  const resultData = results[preferredLang];
+  if (!resultData) {
+    return <div>Error: Language not found.</div>;
+  }
+
+  const { body, src, alt, size, lang } = resultData;
+
   return (
-    <div className="flex flex-col w-full max-w-4xl mx-auto text-center md:mr-12">
+    <div className="flex flex-col w-full text-center">
       <SkillifyNavbar onBackClick={onBackClick} hidden={false} />
       <div className="mt-4 text-2xl font-bold text-black-600">YOUR RESULTS</div>
-      <div className="text-lg font-semibolds px-3">
-        The first coding language you should learn is...
-      </div>
-      <img
-        src={"/images/quiz/languages-quiz/javascript.png"}
-        alt="JavaScript"
-        className=" scale-75"
-      />
-      <div className="text-3xl font-bold text-brandPrimary">JavaScript</div>
+      <div className="text-lg font-semibolds px-3">{body}</div>
+      <img src={src} alt={alt} className={size} />
```

Comment:
would be better if we could just hardcode scale-75 here or have a consistent tailwind classname for all quiz results

Reply:
okay - sounds good!